index.js

```diff
@@ -0,0 +1,93 @@
+// url to retrieve all trips
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const reportStatus = (message) => {
+  $('status-message').html(message);
```

Comment:
This selector `$('status-message')` looks for an element `<status-message>` ... if you wanted the ID of that, you'd use `$('#status-message')`